raft/rafttest/node.go

```diff
@@ -62,6 +64,22 @@ func (n *node) start() {
 				n.Node = nil
 				close(n.stopc)
 				return
+			case p := <-n.pausec:
+				recvms := make([]raftpb.Message, 0)
+				for p {
+					// TODO: locking around paused?
+					n.paused = true
+					select {
+					case m := <-n.iface.recv():
```

Comment:
i do worry about buffer overflow. then the network will begin to drop messages and it breaks the api guarantees.